packages/core/database/lib/entity-manager/regular-relations.js

```diff
@@ -222,8 +222,10 @@ const cleanOrderColumns = async ({ id, attribute, db, inverseRelIds, transaction
   const updateOrderColumn = async () => {
     if (!hasOrderColumn(attribute) || !id) return;
 
+    const joinTableName = addSchema(joinTable.name);
```

Comment:
I tested with mysql 8 and it still works, but it makes me nervous we're potentially changing something in that query that was already working fine. Could we keep this line where it was before just to be sure it doesn't break anything?

Reply:
sure, no prob.

It shouldnt be a problem , as there is no schema defined for mysql . But it's true that it could be an issue. Will change it